draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1641,6 +1641,22 @@ corresponding curve.
 See {{security-considerations-targets}} for more details, and
 {{new-suite}} for guidelines on choosing k for a given curve.
 
+The hash\_to\_field function described below may not be most efficient
+for all fields. For example, when hashing to an element of the extension
+field GF(p^m), hash\_to\_field requires expanding msg into m * L bytes.
+For extension fields where log2(p) is significantly smaller than the security
+level k, this approach is inefficient: it requires expand\_message to output
+roughly m * log2(p) + m * k bits, whereas m * log2(p) + k bytes suffices to
+generate an element of GF(p^m) with bias at most 2^-k. In such cases, an
+applications MAY use an alternative hash\_to\_field function, provided it
+meets the following security requirements:
+
+- The function output MUST be a uniformly random field element.
```

Comment:
```suggestion
- The function MUST output field element(s) that are uniformly random except with bias at most 2^-k.
```

Reply:
The issue is, we've just gotten through saying that the function in this section outputs something with bias at most 2^-k---so it does not even meet the "uniformly random" requirement as stated here!

Reply:
👍 thanks -- this precision is better!